src/db.js

```diff
@@ -15,7 +15,81 @@ db.on('connect', () => {
     console.log('database connected');
 });
 
+// Basic operations
```

Comment:
i like this, but not need use class or aditional file for this, in future here we keep basic functions and helpers we import from separated file, see
```
- db
-- common.js (basic operations)
- - index.js (init db, and re-export from other files)
- - getUser.js
- - ...
```

Reply:
When the future starts?